dali/operators/generic/reduce/reduce.cc

```diff
@@ -30,14 +34,53 @@ Not providing any axis results in reduction of all elements.)code",
     "If True, maintains original input dimensions.",
     false);
 
-DALI_SCHEMA(reductions__Sum)
-  .DocStr("Gets sum of elements along provided axes.")
+DALI_SCHEMA(ReduceWithOutputType)
   .AddOptionalArg("dtype",
     R"code(Output data type. This type is used to accumulate the result.)code",
     DALI_NO_TYPE)
+  .AddParent("ReduceBase");
+
+  DALI_SCHEMA(ReduceWithMeanInput)
+  .AddOptionalArg("ddof",
+    R"code(Delat Degrees of Freedom. Adjusts the divisor used in calculations, which is `N - ddof`.)code",
+    0)
+  .AddParent("ReduceWithOutputType");
+
+DALI_SCHEMA(reductions__Std)
+  .DocStr("Gets standard deviation of elements along provided axes.")
+  .NumInput(2)
+  .NumOutput(1)
+  .AddParent("ReduceWithMeanInput");
+
+DALI_SCHEMA(reductions__Variance)
+  .DocStr("Gets variance of elements along provided axes.")
+  .NumInput(2)
+  .NumOutput(1)
+  .AddParent("ReduceWithMeanInput");
+
+DALI_SCHEMA(reductions__Mean)
+  .DocStr("Gets mean of elements along provided axes.")
   .NumInput(1)
   .NumOutput(1)
-  .AddParent("ReduceBase");
+  .AddParent("ReduceWithOutputType");
+
+DALI_SCHEMA(reductions__MeanSquare)
+  .DocStr("Gets mean square of elements along provided axes.")
+  .NumInput(1)
+  .NumOutput(1)
+  .AddParent("ReduceWithOutputType");
+
+DALI_SCHEMA(reductions__RootMeanSquare)
```

Comment:
```suggestion
DALI_SCHEMA(reductions__RMS)
```
This is a well established term in the field. Also, if we keep it uppercase here it will convert to well to "rms" in fn.

Reply:
Done